client/src/main/java/io/pravega/client/control/impl/ControllerImpl.java

```diff
@@ -1508,6 +1511,31 @@ private TxnSegments convert(CreateTxnResponse response) {
                 });
     }
 
+    @Override
+    public AsyncIterator<io.pravega.controller.stream.api.grpc.v1.Controller.TxnResponse> listCompletedTransactions(Stream stream) {
```

Comment:
`io.pravega.controller.stream.api.grpc.v1.Controller.TxnResponse` can be imported

Reply:
Shouldn't `Stream` be final?

Reply:
Done as suggested.